randovania/games/prime1/exporter/patch_data_factory.py

```diff
@@ -755,6 +755,26 @@ def create_game_specific_data(self) -> dict:
                 for area in region.areas:
                     level_data[region.name]["rooms"][area.name]["submerge"] = self.rng.random() < probability
 
+        # Replace vanilla missile blast shields with the new ones
+        if not self.configuration.legacy_mode:
+            for region in regions:
+                for area in region.areas:
+                    for node in area.nodes:
```

Comment:
Use region_list.iterate_nodes() instead.

Reply:
While iterating through each node from just that way, there was not (at least from what I was testing) an easy way to pull the region and area values from the nodes to make the adjustment to the leveldata

Reply:
region_list.region_with_name(node.identifier.region)
region_list.area_by_name(node.identifier.area)

Reply:
you could even use `region_list.nodes_to_region/area(node)`

Reply:
As discussed in discord, used 

```
world_name = node.identifier.region
room_name = node.identifier.area
```